test/master_test.js

```diff
@@ -38,6 +39,18 @@ describe('Master tests', function() {
         nock.restore();
     });
     beforeEach(function(){
+        nock('https://management.azure.com:443', {'encodedQueryParams':true})
+        .get(/subscriptions$/, /.*/)
+        .query(true)
+        .times(10)
+        .reply(200, mock.subscriptionsResponse);
```

Comment:
Where is this endpoint called?

Reply:
I have found that it gets called in almost every test. I'm not quite sure why.

Reply:
actually it looks like this can eb taken out. so that a bonus.